script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -22,15 +22,15 @@ I'll ask her about their relationship in more detail.
 -- Page 521, Offset 17866.
 -- 「地味で悪かったな。何事も基礎からだって言ったの、おまえだろ」
 // 基礎 translated as fundamentals in D4 whiteboard sesh
-"Well sorry for starting with the basics.
+"Well sorry for asking about something so mundane. 
```

Comment:
I know it's the same JP word, but the en repetition doesn't do it for me here. Feel like if you were to use mundane it would need to be in a more sarcastically constructed sentence, basics is more defensive and fits with the tone more as I read it